gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/FileManager.java

```diff
@@ -217,4 +218,18 @@ private String getFileNameFromUrl(String stringUrl) {
     public String generatePresignedUrl(String minioUrl) {
         return minioAdapter.generatePreSignedUrlFromFullMinioPath(minioUrl, 1);
     }
+
+    public void uploadFileToMinio(OffsetDateTime timestamp, MultipartFile file, String fileType, String fileName) {
+        String processTag = taskManagerConfigurationProperties.getProcess().getTag();
```

Comment:
Maybe we can add some validation of the inputs. At least requireNonNull

Reply:
I asked already about this and in ftp there is only file name validation .... and null doesn't happen